content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md

```diff
@@ -351,7 +352,7 @@ registries:
 
 ### Python
 
-Supported by Artifactory, Azure Artifacts, and Nexus. The {% data variables.product.prodname_registry %} registry is not supported.
+Supported by Artifactory, Azure Artifacts, Nexus, and ProGet. The {% data variables.product.prodname_registry %} registry is not supported.
```

Comment:
@whatatripp it doesn't seem like ProGet supports Python packages from reading the docs right?

```suggestion
```

Reply:
// same (Python feeds) - example added :)